endpoint/endpoint.go

```diff
@@ -107,6 +107,9 @@ func (t Targets) IsLess(o Targets) bool {
 	return false
 }
 
+// ProviderSpecific is a
```

Comment:
Comment is incomplete. Can you please fix it.

Reply:
Done.